Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,14 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            const telemetryProps: { [key: string]: any; } = { MiniMatchLines: sourceFilter.length };
+            var start = new Date().getTime();
             const sources = tl.findMatch(this.inputDataContract.TestSelectionSettings.SearchFolder, sourceFilter);
             tl.debug('tl match count :' + sources.length);
+            var timeTaken = new Date().getTime() - start;
+            tl.debug('Time taken for applying the minimatch pattern to filter out the sources ${timeTaken} ms '+timeTaken);
```

Comment:
this is no longer needed (+timeTaken)

the ${} takes care of printing the variable

Also use "`" the symbol present with the tilde ~ on the keyboard for string interpolation